share/cmake/modules/FindExtPackages.cmake

```diff
@@ -105,20 +89,37 @@ if(OCIO_BUILD_PYTHON OR OCIO_BUILD_DOCS)
     endif()
 endif()
 
-# The presence of OpenImageIO allows additional OCIO apps and the OSL
-# translation unit tests to be built.
+# The presence of OpenImageIO or OpenEXR allows additional OCIO apps to be built.
 
 if(OCIO_BUILD_APPS OR OCIO_BUILD_TESTS)
+    # Always look for OpenImageIO as it is needed for OSL unit tests.
+
     # OpenImageIO
     # https://github.com/OpenImageIO/oiio
-
     set(OIIO_VERSION "2.1.9")
```

Comment:
In the spirit of further cleanup for OCIO 2.2.x, would that be a good time to scrap our custom FindOpenImageIO.cmake script and potentially also increase the minimum OIIO version? OIIO will be used for OSL unit tests and for apps if explicitly asked for. Any thoughts @lgritz?

Reply:
This seems to generate further conflict between our in build Imath and system Imath OIIO is looking for. This will need further investigation. Note that even with the current Find CMake script, the change to OpenEXR by default also introduced some [break](https://github.com/AcademySoftwareFoundation/OpenColorIO/runs/6737918794?check_suite_focus=true#step:8:930) with conflict between OIIO / OSL / OpenEXR.

Reply:
I'm reverting the changes and will try to resolve the Half header conflict mentioned above.